src/components/pages/home/features/features.tsx

```diff
@@ -56,16 +54,18 @@ const featureList: Feature[] = [
 
 function FeatureComponent(props: Feature) {
   return (
-    <div>
-      <div className={'flex flex-nowrap gap-4 items-center mb-2'}>
-        <div className={'rounded-xl bg-gray-100 dark:bg-zinc-800 p-3 pb-0.5 aspect-square'}>
-          <props.icon size={40} />
+    <div className="group p-4 rounded-2xl transition-all duration-300 hover:bg-gray-50 dark:hover:bg-zinc-800/50">
```

Comment:
IMO it does not really make sense that those cards have a hover effect, for me that would indicate that you can click them, but you can't

Reply:
Makes sense, I'll remove it